src/main.c

```diff
@@ -377,7 +369,9 @@ HTTPDirFS options:\n\
         --cache             Enable cache (default: off)\n\
         --cache-location    Set a custom cache location\n\
                             (default: \"${XDG_CACHE_HOME}/httpdirfs\")\n\
-        --cache-clear       Clear cache directory and exit\n\
+        --clear-cache       Delete the cache directory or the custom location\n\
```

Comment:
I prefer to stick with ``--cache-clear``, so you are grouping similar functions by name.